qutebrowser/utils/log.py

```diff
@@ -199,6 +199,9 @@ def init_log(args):
         root.addHandler(console)
     if ram is not None:
         root.addHandler(ram)
+    else:
+        # If we add no handler, we shouldn't process any logs at all.
+        logging.disable(logging.CRITICAL)
```

Comment:
We still (usually) have the console handler. Why not pass `numeric_level` here? That way, doing `--loglines 0 --debug` would still work as expected (and log debug output to the console), and the default would be to still log error/warning messages to the console.